build.gradle

```diff
@@ -42,31 +43,31 @@ deploy {
 // Set this to true to enable desktop support.
 def includeDesktopSupport = true
 
-// Vendor dependencies to ignore when in sim
-// Dependency names found in their respective Json file under the key "name"
-String[] ignoreDeps = ["REVRobotics"]
-
 // Defining my dependencies. In this case, WPILib (+ friends), and vendor libraries.
 // Also defines JUnit 4.
 dependencies {
-    def shouldIgnoreDeps = Os.isFamily(Os.FAMILY_MAC) && (
-        gradle.startParameter.taskNames.toString() =~ /simulate\w+/
-    ) 
+    // Vendor dependencies to ignore when (in sim AND on macos)
+    // OR build is run with -Pignore-deps as an argument
+    // Dependency names found in their respective Json file under the key "name"
+    String[] ignoredDeps = ["REVRobotics", "CTRE-Phoenix"]
+    def shouldIgnoreDeps = project.hasProperty('ignore-deps') || (
+        Os.isFamily(Os.FAMILY_MAC) && (gradle.startParameter.taskNames.toString() =~ /simulate\w+/)

```

Comment:
This has nothing to do with your work but I just wanted to say that the capitalization of `Os` in gradle files is absolutely fetid

Reply:
Yeah, once I saw that I almost gave up trying to make this gradle build work.